src/qn/flux.jl

```diff
@@ -7,47 +6,75 @@ Returns the flux of the ITensor.
 
 If the ITensor is empty or it has no QNs, returns `nothing`.
 """
-function flux(T::ITensor)
-  return flux(tensor(T))
-end
+flux(T::ITensor, args...) = flux(tensor(T), args...)
 
-function checkflux(T::ITensor, flux_check)
-  return checkflux(tensor(T), flux_check)
-end
+"""
+    checkflux(T::ITensor)
 
-function checkflux(T::ITensor)
-  return checkflux(tensor(T))
-end
+Check that fluxes of all non-zero blocks of a blocked or symmetric ITensor
+are equal. Throws an error if one or more blocks have a different flux.
+"""
+checkflux(T::ITensor, flux_check) = checkflux(tensor(T), flux_check)
+
+"""
+    checkflux(T::ITensor, flux)
+
+Check that fluxes of all non-zero blocks of a blocked or symmetric Tensor
+equal the value `flux`. Throws an error if one or more blocks does not have this flux.
+"""
+checkflux(T::ITensor) = checkflux(tensor(T))
 
 #
 # Tensor versions
 # TODO: Move to NDTensors when QN functionality
 # is moved there.
 #
 
-flux(T::Tensor, args...) = flux(inds(T), args...)
+"""
+    flux(T::Tensor, block::Block)
+
+Compute the flux of a specific block of a Tensor, 
+regardless of whether this block is present or not in the storage.
+"""
+flux(T::Tensor, block::Block) = flux(inds(T), block)
+
+"""
+    flux(T::Tensor, i::Integer, is::Integer...)
+
+Compute the flux of a specific element of a Tensor, 
+regardless of whether this element is zero or non-zero.
+"""
+flux(T::Tensor, i::Integer, is::Integer...) = flux(inds(T), i, is...)
+
```

Comment:
For the above two functions, I replaced `flux(T::Tensor, args...) = ` with explicit specializations for the case of Block and Integer arguments. All tests pass so these must be the only two cases we are using.